src/components/autocomplete/js/autocompleteController.js

```diff
@@ -639,7 +679,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
   /**
    * Clears the searchText value and selected item.
    */
-  function clearValue ($event) {
+  function clearValue (shouldClearText) {
```

Comment:
Agreed, I can see the point. Thanks for the clarification - Probably rename the boolean to
```js
function clearValue(clearText) {}
```